emulation/setup.py

```diff
@@ -5,18 +5,7 @@
 
```

Comment:
I'd consider putting this `emulation` subdir in the tests, since we don't plan to distribute it.